backend/src/main/kotlin/app/ehrenamtskarte/backend/cards/database/Schema.kt

```diff
@@ -36,6 +37,7 @@ object Cards : IntIdTable() {
     val cardInfoHash = binary("cardInfoHash", CARD_INFO_HASH_LENGTH).uniqueIndex()
     val codeType = enumeration("codeType", CodeType::class)
     val firstActivationDate = timestamp("firstActivationDate").nullable()
+    val entitlementsId = reference("entitlementsId", UserEntitlements).nullable()
```

Comment:
:question: Is the plural `s` correct here?

Reply:
in my head it was, but I think you are right

Reply:
changed to singular